src/components/delete-account/index.njk

```diff
@@ -12,13 +12,26 @@
 <h1 class="govuk-heading-l govuk-!-margin-top-0">{{'pages.deleteAccount.header' | translate}}</h1>
 
 <p class="govuk-body">{{'pages.deleteAccount.paragraph1' | translate}}</p>
-<p class="govuk-body">{{ 'pages.deleteAccount.paragraph2' | translate }}</p>
-<p class="govuk-body">{{ 'pages.deleteAccount.paragraph3' 
-    | translate
-    | replace("[emailManagementLink]", manageEmailsLink)
-    | safe
-    }}</p>
-<p class="govuk-body">{{ 'pages.deleteAccount.paragraph4' | translate }}</p>
+
+ {% if services.length %}
+  <p class="govuk-body">{{ 'pages.deleteAccount.paragraph2' | translate }}</p>
+  <ul class="govuk-list govuk-list--bullet">
+    {% for service in services %}
+
+      {% set locale = ['pages.yourServices.clientRegistry.', env, '.', service.client_id, '.'] | join %}
+        <li id="service-list-item">{{ [locale, 'header'] | join | translate }}</li>
+
+    {% endfor %}
+    </ul>
+    {% if hasGovUkEmailSubscription %}
+    <p class="govuk-body" id="govuk-email-subscription-info">{{ 'pages.deleteAccount.paragraph3' | translate }}</p>
```

Comment:
Nitpick: we can tidy up the indentation in this template to make it a little more readable